Change ariaTSsetup input to text file

ehavazli opened the pull request:
Added the functionality to write product paths to a text file and changed the `ariaTSsetup.py` input to text file.
Previous method of using wildcards is preserved but commented out.

EJFielding commented:
Hi @ehavazli, Can you look at the changes that are in @katia-tymofyeyeva  PR #69 ? She modified the `ariaTSsetup` call to use a date selection section that Adrian wrote. It should be possible to do the text file creation that you are suggesting but also do the date selection. It seems that `ariaTSsetup` does not have its own date selection like `ariaDownload` has.

ehavazli commented:
> Hi @ehavazli, Can you look at the changes that are in @katia-tymofyeyeva PR #69 ? She modified the `ariaTSsetup` call to use a date selection section that Adrian wrote. It should be possible to do the text file creation that you are suggesting but also do the date selection. It seems that `ariaTSsetup` does not have its own date selection like `ariaDownload` has.

Thank you for pointing out that PR @EJFielding, I am going to update this PR after @katia-tymofyeyeva merges that one.

ehavazli commented:
@EJFielding @rzinke @katia-tymofyeyeva 
I incorporated the final changes from Adrian to this PR and changed the `-nt 16` to `-nt 8` just to be sure that the user has enough CPU's available.
